cloud: release notes 20230718 (#14260)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## July 18, 2023

**General changes**

> - Support role-based access control at both the organization level and project level, which lets you grant roles with minimum permissions to users for better security, compliance, and productivity.

Reviewer: ```suggestion
- Refine role-based access control at both the organization level and project level, which lets you grant roles with minimum permissions to users for better security, compliance, and productivity.
```

    - The organization roles include `Organization Owner`, `Organization Billing Admin`, `Organization Console Audit Admin`, and `Organization Member`.
    - The project roles include `Project Owner`, `Project Data Access Read-Write`, and `Project Data Access Read-Only`.
    - To manage clusters in a project (such as cluster creation, modification, and deletion), you need to be in the `Organization Owner` or `Project Owner` role. 
  
  For more information about permissions of different roles, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).

> - The Customer-Managed Encryption Key (CMEK) feature is now generally available (GA) for TiDB Dedicated clusters hosted on AWS.

Reviewer: ```suggestion
- Support the Customer-Managed Encryption Key (CMEK) feature (beta) for TiDB Dedicated clusters hosted on AWS.
```

  
    You can create CMEK based on AWS KMS to encrypt data stored in EBS and S3 directly from the TiDB Cloud console. This ensures that customer data is encrypted with a key managed by the customer, which enhances security.
  

>     Note that this feature still has restrictions and is only available upon request. To apply for this feature, contact [TiDB Cloud support](/tidb-cloud/tidb-cloud-support.md).

Reviewer: ```suggestion
    Note that this feature still has restrictions and is only available upon request. To apply for this feature, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```
